FeedStoreChallenge/Infrastructure/ManagedFeedImage.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  ManagedFeedImage.swift
+//  FeedStoreChallenge
+//
+//  Created by Fernando Goulart on 01/08/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import CoreData
+
+@objc(ManagedFeedImage)
+class ManagedFeedImage: NSManagedObject {
```

Comment:
This class is not intended to be inherited thus it should be `final`

Please make the class `final`.

```suggestion
final class ManagedFeedImage: NSManagedObject {
```